packages/xchain-aggregator/src/types.ts

```diff
@@ -14,9 +23,9 @@ type TxSubmitted = {
  * Fees
  */
 type Fees = {
-  asset: Asset | TokenAsset | SynthAsset // The asset for which fees are calculated
-  affiliateFee: CryptoAmount<Asset | TokenAsset | SynthAsset> // The affiliate fee amount
-  outboundFee: CryptoAmount<Asset | TokenAsset | SynthAsset> // The outbound fee amount
+  asset: Asset | TokenAsset | SynthAsset | TradeAsset // The asset for which fees are calculated
+  affiliateFee: CryptoAmount<Asset | TokenAsset | SynthAsset | TradeAsset> // The affiliate fee amount
+  outboundFee: CryptoAmount<Asset | TokenAsset | SynthAsset | TradeAsset> // The outbound fee amount
```

Comment:
Outbound fee won't be a trade asset. Trade Assets live within THORChain, even with Order books.

Reply:
there's any problem if i do this change?

Reply:
Looking at [documentation](https://thornode.ninerealms.com/thorchain/doc), in the QuoteFees type, the asset is the one used for all fees. If you are right, and outBoundFee is the native asset fee of the target fee. The bug is already there, we can create a new issue to fix